dev-resources/Documents/Git Sequence Flow.txt

```diff
@@ -0,0 +1,12 @@
+Git Sequence Flow:
+
+1. Make changes in local feature branch (dev-pom-update-local-fixes-and-docs-1).
```

Comment:
don't document the branch in here, because that's no applicable. Actually in general this flow is true of ALL projects, so this is a process note you should probably just keep but doesnt go in the repo.